terraform/cloudwatch.tf.json.template.py

```diff
@@ -16,6 +22,21 @@
 )
 
 
+def alarm_resource_name(threshold: MetricThreshold) -> str:
+    # FIXME: Remove this hack of slicing off the prefix from the handler name
+    #        once the Lambda's have been renamed.
+    #        https://github.com/DataBiosphere/azul/issues/5337
+    handler_name = threshold.handler_name
+    if handler_name and handler_name.startswith(threshold.lambda_name):
```

Comment:
We don't conflate `None` and empty string. One is expected, the other is an error.